### DISCUSSION TITLE
wine64-preloader claims usb interface for no reason

### DISCUSSION THREAD

**Discussion by Cryptiiiic** [The-Wineskin-Project/WineskinServer#280] [Q&A] [answered]
wine64-preloader claims usb interface for no reason. I can't do anything usb related to when wine is running because my usb interface is claimed :(
`<Error>: Unable to open device: e00002c5`
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/27748705/180115624-57257404-9f84-4015-9b14-7d893e9f7749.png">

**Comment by Gcenx** [comment, marked as answer]
> wine64-preloader claims usb interface for no reason. 

It’s not for no reason, modern wine does indeed access the usb interfaces hence why it’s built with libusb.

You can set `wineusb` to disabled in winecfg that will make wine cry in its logs but should stop your usb devices from being claimed.

![Screen Shot 2022-07-22 at 8 12 43 AM](https://user-images.githubusercontent.com/38226388/180437600-34be9b53-a37f-4ec1-8551-faad31945daf.png)

**Comment by Cryptiiiic** [comment]
I didn't see wineusb in winecfg or figure that part out so I just renamed all the files.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/27748705/180310818-bbe1ce64-2409-4695-8455-b37e9779c137.png">

**Comment by Gcenx** [comment]
![Screen Shot 2022-07-22 at 8 12 43 AM](https://user-images.githubusercontent.com/38226388/180436936-430998e4-b259-4eef-a331-e551be371439.png)

As can be seen in this screenshot you'd just manually type in `wineusb` then click yes to the warning, next set to disabled.
